src/cldr.js

```diff
@@ -4,7 +4,7 @@
 var expDTComponents = /(?:[Eec]{1,6}|G{1,5}|(?:[yYu]+|U{1,5})|[ML]{1,5}|d{1,2}|a|[hkHK]{1,2}|m{1,2}|s{1,2}|z{1,4})(?=([^']*'[^']*')*[^']*$)/g;
 
 // Skip over patterns with these datetime components
-var unwantedDTCs = /[QxXVOvZASjgFDwWIQqH]/;
```

Comment:
Not sure why I did that, maybe I misread the spec or misinterpreted something.  Just had another quick look at the spec and it probably shouldn't have been there.